addons/mail/static/src/core/common/discuss_app_category_model.js

```diff
@@ -22,7 +23,10 @@ export class DiscussAppCategory extends Record {
             return String.prototype.localeCompare.call(t1.name, t2.name);
         }
         if (this.id === "chats") {
-            return t2.lastInterestDateTime.ts - t1.lastInterestDateTime.ts;
+            return compareDatetime(t1, t2, "lastInterestDateTime", {
```

Comment:
I'm also not especially a fan of the function accessing the field, I think we should just provide date1 and date2 (or undefineds)